src/buffer/out/textBuffer.cpp

```diff
@@ -441,11 +441,16 @@ void TextBuffer::_PrepareForDoubleByteSequence(const DbcsAttribute dbcsAttribute
     }
 }
 
-void TextBuffer::ConsumeGrapheme(std::wstring_view& chars) noexcept
+size_t TextBuffer::GraphemeNext(const std::wstring_view& chars, size_t position) noexcept

```

Comment:
Now, what is `position`. A column or a string index?

Reply:
Ah, good point. I forgot to add function documentation. 🙂
(It's a string index, since it's for text-based navigation.)